tensorflow/contrib/ffmpeg/default/ffmpeg_lib.cc

```diff
@@ -208,22 +311,23 @@ string GetTempFilename(const string& extension) {
     }
     struct stat statbuf;
     if (!stat(dir, &statbuf) && S_ISDIR(statbuf.st_mode)) {
-      return io::JoinPath(dir, StrCat("tmp_file_", getpid(), ".", extension));
+      if (extension.length()) {
+        return io::JoinPath(dir, StrCat("tmp_file_", getpid(), ".", extension));
```

Comment:
This is not going to work well for multi-threaded access.

Reply:
Please see #13394

Reply:
Thanks. Done.